### PR TITLE
Change the instructions to comments for the issue template

### PR DESCRIPTION
**Pull request by jverbraeken**
- Relevant Issues: -
- Related Pull Requests: -

* * *

## What
This PR adds a simpler to use issue template by making commenting irrelevant text and by giving the first few words of important sentences.

## Why
This PR is needed to make the life of the programmer a bit easier.

## How
This feature cannot be viewed in the project because it is not part of the project.

## Alternative implementation
-

## Notes
-

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -9,16 +9,22 @@
 The user story for the issue.
 
 ## What
-Specify in more detail what this issue adds to the repository.
+This issue demands 
+[//]: # "Specify in more detail what this issue adds to the repository."
 
 ## Why
-Specify in more detail why this issue is needed in the project.
+This issue is need for the project because
```
Comment (ericcornelissen) [inline ISSUE_TEMPLATE.md]: > This issue is needed* for the project because

**Comment by jverbraeken** [comment]
@ericcornelissen Very good suggestion! I really should have looked at the file to check if it was formatted correctly. It's fixed now

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -9,16 +9,22 @@
...
-Specify in general terms how this feature can be added.
+This feature can be added by 
+<!-- Specify in general terms how this feature can be added. -->
 
 ## Definition of done
-Specify exactly what must be done before this task can be considered done.
+This task can be considered done when
+
+- 
+<!-- Specify exactly what must be done before this task can be considered done. -->
```
Comment (clenrock) [inline ISSUE_TEMPLATE.md]: Why are these enters here?

Comment (jverbraeken) [inline ISSUE_TEMPLATE.md]: @clenrock Because Github won't recognize it as a list otherwise, for some reason

Comment (ericcornelissen) [inline ISSUE_TEMPLATE.md]: Ah, I missed the `-` on line 26 (this is true for Markdown in general fyi)

**Pull request merged**